src/pypgstac/pyproject.toml

```diff
@@ -0,0 +1,129 @@
+[project]
+name = "pypgstac"
+description = "Schema, functions and a python library for storing and accessing STAC collections and items in PostgreSQL"
+readme = "README.md"
+requires-python = ">=3.8"
+license = "MIT"
+authors = [
+    {name = "David Bitner", email = "bitner@dbspatial.com"},
+]
+keywords = ["STAC", "Postgresql", "PgSTAC"]
+classifiers = [
+    "Intended Audience :: Developers",
+    "Intended Audience :: Information Technology",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+]
+dynamic = ["version"]
+dependencies = [
+    "smart-open>=4.2,<7.0",
+    "orjson>=3.5.2",
+    "python-dateutil==2.8.*",
+    "fire==0.4.*",
+    "plpygis==0.2.*",
+    "pydantic[dotenv]==1.10.*",
+    "tenacity==8.1.*",
+    "methodtools==0.4.5",
+]
+
+[project.optional-dependencies]
+test = [
+    "pytest",
+    "pytest-cov",
+    "pystac[validation]==1.*",
+]
+dev = [
+    "flake8==3.9.*",
+    "black>=21.7b0",
+    "mypy>=0.910",
+    "types-orjson==0.1.1",
+    "ruff==0.0.225",
```

Comment:
pre-commit ruff version is not the same https://github.com/stac-utils/pgstac/pull/155/files#diff-63a9c44a44acf85fea213a857769990937107cf072831e1a26808cfde9d096b9R19

Could we just use `pre-commit` directly without having to set a dev dependency?